src/components/Search/SearchResultsList.tsx

```diff
@@ -4,12 +4,13 @@ import Box from '@mui/material/Box';
 import List from '@mui/material/List';
 import { connect } from 'react-redux';
 
+import useCheckRevision from '../../hooks/useCheckRevision';
```

Comment:
Not for this PR, and not sure if you've talked about this before. But I feel like this `hooks` directory will be pretty crowded in the future when we keep adding new hooks to the codebase. I wonder if we can make that a bit better by grouping the related hooks together. Like with a similar structure to the `components` directory. What do you think?

Reply:
that's a good idea :) filed #126